README.adoc

```diff
@@ -1,40 +1,99 @@
 
-GitHub doesn’t (yet) whitelist the AsciiDoc plugin, so you can only run it on your own machine… or use a continuous integration platform like https://travis-ci.org/[Travis CI].
+= A Note About GitHub Pages
 
-You need two branches in your repository:
+GitHub Pages does not (yet) whitelist the jekyll-asciidoc plug-in, so you can not write `.adoc` posts and have them instantly publish like Markdown posts do. 
 
-* **master** for markup sources and configuration (may be named differently),
-* **gh-pages** for generated static content (will be created automatically for you).
+You can help change this by creating a support case through http://github.com/support and telling the GitHub team that you want the choice to write in AsciiDoc and have it handled the same as Markdown when pushed to your GitHub Page. Your voice counts: make it heard!
 
-The goal is to configure Travis CI job to listen for commits on the _master_ branch, automatically run jekyll build and push the generated content to the _gh-pages_ branch.
+== So what does that mean?
 
-. Install travis gem:
+Unlike some "fork and write" repos that exist, you need to initially configure the repo with a computer to publish using AsciiDoc. After initially configuring the repo, you can use Git command-line, or even a Git client on your tablet or smartphone to write, commit, and automatically publish blog posts.
+
+This repo uses the https://travis-ci.org/[Travis CI] Continuous Integration (CI) server to emulate GitHub Pages staging automation, and pushes your blog live upon committing any change to the repo. 
+
+= Repo Structure
+
+The repo requires the following structure to work correctly:
+
+* **master**, for markup sources and configuration. This branch can be named anything you choose, however **master** is a general standard used in Jekyll blogs.
+* **gh-pages**, for the generated static content produced by Travis CI. This branch is the username.github.io GitHub Pages domain, which is created automatically for you when the Travis CI job runs.
+
+= Configuring the Repo for Publishing
+
+The goal of this procedure set is to configure a Travis CI job to listen for commits on the _master_ branch, automatically run the jekyll build, and push the generated content to the _gh-pages_ branch.
+
+== Install Minimum Jekyll Requirements
+
+You must install some software to execute commands in subsequent procedures. Meet the Requirements on the http://jekyllrb.com/docs/installation/[Jekyll Installation] page, which link to pages describing how to install: 
+* ruby
+* rubygems
+
+For yum-based package managers, the command to run is:
++
+		$ sudo yum install ruby rubygems
+
+== Install Travis Gem
+
+When you install rubygems, you can use the gem internal package management system to install the Travis CI gem. This gem contains tools required to encrypt GitHub Keys, which is the only safe way of doing this.
```

Comment:
Have changed to:

This gem contains--among other things--a command-line tool for easily encrypting GitHub tokens.